multidict/_multilib/pair_list.h

```diff
@@ -53,8 +53,22 @@ typedef struct pair_list {
  * It is incremented each time that a dictionary is created and each
  * time that a dictionary is modified. */
 static uint64_t pair_list_global_version = 0;
+#ifdef Py_GIL_DISABLED
+static PyMutex global_version_mutex;
+#endif
 
+#ifdef Py_GIL_DISABLED
+static inline
+uint64_t NEXT_VERSION(void)
+{
+    PyMutex_Lock(&global_version_mutex);
```

Comment:
I think that the mutex is not needed here.

Straight loose increment is enough, the version number is an auxiliary thing.
If the global counter slightly desynchronized between threads for short time period -- it is ok, no existing code is broken.

Reply:
Mutex removed. Thanks for the help here!